docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# base Dockerfile: PyTorch nightly docker
+# ghcr.io/pytorch:pytorch-nightly
+FROM ghcr.io/pytorch/pytorch-nightly:latest
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+
+# setup conda by default
+RUN ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
+    echo ". /opt/conda/etc/profile.d/conda.sh" >> ${HOME}/.bashrc
+
+RUN apt-get update
+RUN apt-get install -y git git-lfs jq
+
+RUN git clone https://github.com/pytorch/functorch /workspace/functorch
+RUN git clone https://github.com/pytorch/torchdynamo /workspace/torchdynamo
+RUN git clone https://github.com/pytorch/benchmark /workspace/benchmark
+
+# Clone conda env
+RUN conda create --name torchbench --clone base && \
+    echo "conda activate torchbench" >> ${HOME}/.bashrc
+
+# Run the setup
+RUN cd /workspace/benchmark; python install.py
+RUN cd /workspace/functorch; python setup.py develop
```

Comment:
To keep dependency versions the same, we can add a `pip uninstall functorch -y` here to remove the existing installations.

Same for `torchdynamo`

Reply:
Adding this should be fine; just wondering in what case there will be existing installations?

Reply:
Hmm, our current pytorch build installs all those libraries for benchmark. I think it's ok to leave it like this for now. We'll modify our docker build to not install them when we use a commonly shared benchmark infrastructure.